test/e2e/e2e_test.go

```diff
@@ -335,26 +316,63 @@ func verifyResources(t *testing.T, awaitility *wait.Awaitility, murName string,
 		assert.Equal(t, userAccount.Name, rb.Subjects[0].Name)
 		assert.Equal(t, "edit", rb.RoleRef.Name)
 	}
+
+	// Get member cluster to verify that it was used to provision user accounts
+	memberCluster, ok, err := hostAwait.GetKubeFedCluster(cluster.Member, nil)
+	require.NoError(t, err)
+	require.True(t, ok)
+
+	// Then finally check again the MasterUserRecord with the expected (embedded) UserAccount status, on top of the other criteria
+	expectedEmbededUaStatus := toolchainv1alpha1.UserAccountStatusEmbedded{
```

Comment:
Embedded, not Embeded

Reply:
Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/51/commits/3a8d5151b0f2a32f5e3cb7821ef9e2a14b9be5f2